Makefile

```diff
@@ -17,6 +17,8 @@ DEFAULT_CLONE_URL := https://github.com/huggingface/optimum-habana.git
 # If CLONE_URL is empty, revert to DEFAULT_CLONE_URL
 REAL_CLONE_URL = $(if $(CLONE_URL),$(CLONE_URL),$(DEFAULT_CLONE_URL))
 
+export PT_HPU_LAZY_MODE=1
+# will be removed when lazy is disabled
```

Comment:
Hmm not sure we want to do that for any command defined in this file. For example, I wouldn't expect `make style` to export such an environment variable.
Is the logic here to make lazy the default when launching tests?

Reply:
yes. Do you have other suggestion?

Reply:
@regisss do you mean to update files such as https://github.com/huggingface/optimum-habana/blob/main/tests/ci/slow_tests_1x.sh to include the env?

Reply:
Yes, or just putting `PT_HPU_LAZY_MODE=1` at the front of all pytest commands in the Makefile